ext/pgsql/pgsql.c

```diff
@@ -558,8 +612,12 @@ PHP_MINIT_FUNCTION(pgsql)
 PHP_MSHUTDOWN_FUNCTION(pgsql)
 {
 	UNREGISTER_INI_ENTRIES();
+	size_t i;
 	zend_hash_destroy(&PGG(connections));
 
+	for (i = 0; i < 11; i ++)
```

Comment:
You can merge the assignment and declaration of `i`, we're in C99.

Reply:
Can you also use a sizeof expression instead of the hardcoded 11? Or alternatively, use a define instead of 11 for both the declaration of the array and this loop?